C/Cairomm/Cairomm@1.14.4/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Cairomm"
+version = v"1.14.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://www.cairographics.org/releases/cairomm-$(version).tar.xz",
+                  "4749d25a2b2ef67cc0c014caaf5c87fa46792fc4b3ede186fb0fc932d2055158")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/cairomm*/
+mkdir output && cd output/
+meson --cross-file=${MESON_TARGET_TOOLCHAIN%.*}_gcc.meson ..
+ninja -j${nproc}
+ninja install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+
+# These platforms are not supported by Cairo_jll yet
+platforms = filter!(p -> arch(p) != "armv6l", supported_platforms())
+platforms = filter!(p -> !(Sys.isapple(p) && arch(p) == "aarch64"), platforms)
+platforms = filter!(p -> libc(p) != "musl", platforms)
```

Comment:
Uhm, only armv6l isn't there (and we mainly just make sure to rebuild all dependencies, nothing fundamental): https://github.com/JuliaPackaging/Yggdrasil/blob/fba5fe9f5c4e78e2dc0c25f479c65ce75111049a/C/Cairo/build_tarballs.jl#L67

Reply:
Yeah, finding which dependencies of Cairo are missing armv6l builds.  I think it's mostly the X11 stack, we never touched that

Reply:
Is it alright if I avoid enabling armv6l right now, and just trigger the rebuild for musl/aarch64-darwin?

I'm still trying to make all the way up to `GTKmm3` on the existing platforms, and I've got a ways to go..

Reply:
I'm ok with forgetting about armv6l, but Cairo_jll should be already available for all the other platforms: https://github.com/JuliaBinaryWrappers/Cairo_jll.jl/blob/5105339972cc9aad07d9ef2760a24e61ebeb4674/README.md#platforms